cli/lsp/repl.rs

```diff
@@ -157,10 +160,19 @@ impl ReplLanguageServer {
           CompletionTextEdit::Edit(edit) => Some(ReplCompletionItem {
             new_text: edit.new_text,
             range: lsp_range_to_std_range(&text_info, &edit.range),
+            sort_text: item.sort_text,
+            label: item.label,
           }),
           CompletionTextEdit::InsertAndReplace(_) => None,
         })
       })
+      .sorted_by_key(|item| {
```

Comment:
I think we could just sort `items` vector in place above? That would allow us to remove the added properties and dependency.

Reply:
It's exactly as you said. I fixed it.